xapian-applications/omega/templates/query

```diff
@@ -52,50 +52,55 @@ document.write("<span>Modified:<time datetime=\""+D+" "+T+"\"><b>"+D+"</b></time
 <body>
 <form name=P method=get
 action="$html{$or{$env{SCRIPT_NAME},omega}}" target="_top">
+<div class="query-container">
 ${Search Control(Start) - shows search button, options and textbox}
-<input id="omega-autofocus" type=search name=P value="$html{$query}" size=65 autofocus>
+<input class="query-textbox" id="omega-autofocus" type=search name=P value="$html{$query}" size=65 autofocus>
```

Comment:
You almost certainly don't need this for CSS styling. `input[type=search]` will probably do, as would `input[name=P]` or for that matter `input#omega-autofocus`.

It isn't needed for purely semantic reasons, as it should be obvious to anyone reading the code (at least the output HTML) what this input does.